src/app/pages/storage/modules/pool-manager/components/pool-manager-wizard/pool-manager-wizard.component.html

```diff
@@ -99,7 +99,7 @@
       </mat-step>
 
       <mat-step
-        *ngIf="!(usesDraidLayout$ | async) && !alreadyHasSpare"
+        *ngIf="!(usesDraidLayout$ | async) && !alreadyHasSpare && state.topology[PoolCreationWizardStep.Spare]"
```

Comment:
I think it's time to extract all of this into `hasSpareStep$` or something similar.